app/src/main/java/de/danoeh/antennapod/fragment/ItunesSearchFragment.java

```diff
@@ -192,7 +192,7 @@ private void loadToplist() {
         progressBar.setVisibility(View.VISIBLE);
 
         ItunesTopListLoader loader = new ItunesTopListLoader(getContext());
-        disposable = loader.loadToplist(25)
+        disposable = loader.loadToplist(100)
```

Comment:
I think something like this should be discussed in #3788. We already cause quite a lot of traffic to iTunes and I want to keep it to a minimum, so we do not get locked out one day.

Reply:
my bad this was from me playing around in #3788 in the same branch, I reverted back to 25